biblio/package.json

```
@@ -0,0 +1,14 @@
+{
+  "name": "TBD",
+  "version": "VERSIONED-DURING-PUBLISH",
+  "description": "Machine-readable representation of the internals of the ecma-262 spec",
+  "keywords": [
+    "ecmascript"
+  ],
+  "author": "TC39",
+  "main": "biblio.json",
```

```suggestion
  "main": "biblio.json",
  "exports": {
    ".": "./biblio.json",
    "./package.json": "./package.json"
  },
```
since it's a potential breaking change to add later :-)